HappyApp.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime
+from time import sleep 
+from config import MessageType, Hour, day, _1PM, _8AM, _9PM
+from functions import isToday, scramble, alert
+from queue import Queue
+import readJSON
+from random import shuffle
+from mail import Mail
+
+class HappyApp:
+    def __init__(self,):
+        # For taking into account a sadir solider who was chosen to solo
+        self.sadirLeftOver = None
```

Comment:
I don't think this should be kept in ram.

Reply:
Also, lower_camel_case.

Reply:
So, what is your suggestion? keep it in the future db? (currently the json file)

Is CamelCase a bad practice? or just a practice not used by our section? I dont see the problem.